src/pages/HomePage.tsx

```diff
@@ -218,6 +222,10 @@ class HomePage extends React.Component<Props, State> {
         return <Blog data={this.state.data}></Blog>
       case 'note':
         return <Note data={this.state.data}></Note>
+      case 'video-archive':
+        return <Archive data={this.props.match.params.videoType} queryType='video'></Archive>
```

Comment:
Why not just pass the `pageType` to `Archive` instead of defining different strings for `queryType`?